webapp/app/js/perf.js

```diff
@@ -82,6 +82,14 @@ perf.factory('PhSeries', ['$http', 'thServiceDomain', function($http, thServiceD
           });
 
         if (seriesSummary) {
+          if (!seriesSummary.subtestSignatures) {
+            seriesSummary = _.find(lists.seriesList,
```

Comment:
looking at this a bit more, I think we might not need that.  A few things:
1) compare view for subtests doesn't support >1 header, it is designed for
1 header and a list of pages.
2) we only want one instance, the one from the graph that we clicked on :)

I am not sure right now how to get the specific data from the graph.  Given
the platform, signature, and revision- is it really likely that we will
have >1 signature?  Most likely not.  What if instead we searched for a
second instance and if there was one we add it to the list of errors or
skipped tests to be displayed?


On Mon, May 4, 2015 at 6:11 PM, William Lachance <notifications@github.com>
wrote:

> In webapp/app/js/perf.js
> <https://github.com/mozilla/treeherder-ui/pull/506#discussion_r29629110>:
>
> > @@ -82,6 +82,14 @@ perf.factory('PhSeries', ['$http', 'thServiceDomain', function($http, thServiceD
> >            });
> >
> >          if (seriesSummary) {
> > +          if (!seriesSummary.subtestSignatures) {
> > +            seriesSummary = _.find(lists.seriesList,
>
> It occurs to me that this won't work if we have multiple suites with the
> same subtests. Not currently the case, but it could happen if we changed a
> suite. So we should allow for multiple results here (and getting the union
> of all subtest signatures from them).
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mozilla/treeherder-ui/pull/506/files#r29629110>.
>